src/components/memoBlock/index.ts

```diff
@@ -2,14 +2,20 @@ import styled from "styled-components";
 import MemoBlock from "./memo.component";
 
 export default styled(MemoBlock)`
-  margin-top: 2rem;
   margin: auto;
   max-width: fit-content;
   display: grid;
   grid-template-columns: repeat(8, 1fr);
   grid-template-rows: repeat(4, 1fr);
   gap: 2px;
+  margin-top: 2rem;
 
+  @media screen and (max-width: 55.6em) {
+    // 55.6em = 890px
```

Comment:
In the previous comment, I meant to place it above the media keyword. Why? Because explanations go above what they explain, not below.

```suggestion
  // 55.6em = 890px
  @media screen and (max-width: 55.6em) {
```